fontforge/python.c

```diff
@@ -12290,6 +12305,8 @@ return( -1 );
 	    BaseFree(base);
 return( -1 );
 	}
+	bs = chunkalloc(sizeof(struct basescript));
+	bs->next = NULL;
```

Comment:
this shouldn't be needed since you just chunkalloc'ed it (which uses calloc)

Reply:
I somehow missed all the chunkalloc calls. I guess I ought to have looked more closely.